### ISSUE TITLE
TypeError: babel.loadOptionsSync is not a function

### ISSUE THREAD

**Issue by wshowair** [skovhus/jest-codemods#235]
First of all, thanks for creating this package, it really helps in migrating to Jest easily.

While I was using the latest version@0.25.0 on one of my private repos, I was facing this error:
```
TypeError: babel.loadOptionsSync is not a function
    at exports.transformSync (node_modules/@babel/register/lib/worker/transform.js:87:24)
    at Function.handleMessage (node_modules/@babel/register/lib/worker/handle-message.js:23:16)
    at LocalClient.<anonymous> (node_modules/@babel/register/lib/worker-client.js:134:86)
    at LocalClient.transform (node_modules/@babel/register/lib/worker-client.js:64:47)
    at compile (node_modules/@babel/register/lib/worker/transform.js:87
    const opts = babel.loadOptionsSync(Object.assign({
```

### Steps to reproduce
1. checkout my sample rep https://github.com/wshowair/test-jest-codemods
2. npx jest-codemods
3. Make the following selections
<img width="1341" alt="image" src="https://user-images.githubusercontent.com/16110284/148002724-51d91a71-68b4-4564-8bc2-2053e0c775c2.png">

### Proposed Solution
Upgrade the dependency  @babel/core to 7.8.0 which introduces the missing function.

**Comment by wshowair** [comment]
I can help by creating the PR if it is needed.

**Comment by skovhus** [comment]
> I can help by creating the PR if it is needed.

That would be great. But I'm wondering why this started failing. 🤔

**Comment by wshowair** [comment]
There are a couple of things contributed to this error to happen just recently:

1. Babel released a new version 7.16.7 few days ago https://github.com/babel/babel/blob/v7.16.7/packages/babel-register/src/worker/transform.js#L81 where the function was first introduced. Please note it matched the callstack reported in the issue description ☝️ 
2. When you use `npx jest-codemods` or `npm install -g jest-codemods`:
  - it pulls `@babel/register@7.16.7` at the moment of writing this comment, why? because `jscodeshift` has a dependency `"@babel/register": "^7.13.16"`
  - it pulls `@babel/core@7.4.5` why? because you have it as a fixed versions dependency `"@babel/core": "7.4.5"`. Obviously this version does not have any definition of the missing function `loadOptionsSync`

I will create the pr to upgrade your dependency to `@babel/core: ^7.13.16` to match exactly the current version of `jscodeshift` dependency
